code/game/machinery/computer/dna_console.dm

```diff
@@ -160,50 +160,52 @@
 		genetic_damage_pulse()
 		return
 
-/obj/machinery/computer/scan_consolenew/attackby(obj/item/I, mob/user, params)
+/obj/machinery/computer/scan_consolenew/attackby(obj/item/Item, mob/user, params)
```

Comment:
We don't ever name variables like this. Sometimes single letter variables are capitialized, but we're trying to phase single letter vars out in general. `item` would do just fine though, and iss a whole lot better then `I`

Reply:
Updated. I think Capitalized was a little more readable but lets just make this right. I also lowercased activator var

Reply:
Well I think that making item starting from low capital breaks the check...